meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -672,6 +675,7 @@ CONFIG_BLK_DEV_BSG_COMMON=y
 # CONFIG_BLK_DEV_BSGLIB is not set
 CONFIG_BLK_DEV_INTEGRITY=y
 CONFIG_BLK_DEV_INTEGRITY_T10=m
+CONFIG_BLK_DEV_WRITE_MOUNTED=y
```

Comment:
Nitpick: It'd be interesting to find out if we actually have userspace that depends on this. If not, disabling it would be a good idea. For now, it's ok though, as it maintains existing behavior.